src/main/java/me/botsko/prism/listeners/PrismPlayerEvents.java

```diff
@@ -324,9 +324,15 @@ public void onPlayerInteract(final PlayerInteractEvent event) {
             // " Item in hand:" + player.getItemInHand().getTypeId() + ":" +
             // player.getItemInHand().getDurability());
 
+            int itemInHandId = player.getItemInHand().getTypeId();
```

Comment:
I'm asking you add comments to the code itself. My intention is to make this clear to future devs who are confused why such a weird check exists. It's documented here, so they'll always find it, but a change this unimportant can easily be commented.

Your code should look like this:

```
// 1.10 and newer bukkit versions treat an empty hand as 0, not -1 like older versions
short itemInHandDurability = player.getItemInHand().getDurability();
if (player.getItemInHand().getTypeId() == 0 && itemInHandDurability == 0) {
    itemInHandDurability = -1;
}
```